### PR TITLE
Add comment to make sure clay bucket recipe is properly updated

### PR DESCRIPTION
**Pull request by D-Cysteine**

### CODE REVIEW

File: scripts/Iguana-Tweaks.zs [hunk 1]
```diff
@@ -24,6 +24,8 @@ recipes.remove(<IguanaTweaksTConstruct:sawManyullyn>);
 
 
 // --- Unfired Clay Bucket
+// If you change this recipe, be sure to update gregtech.api.enums.OrePrefixes!
+// bucketClay.mSecondaryMaterial controls what the clay bucket pulverizes into.
 recipes.addShaped(<IguanaTweaksTConstruct:clayBucketUnfired>, [
 [null, null, null],
 [<ore:dustClay>, null, <ore:dustClay>],
```
Comment (Dream-Master) [inline scripts/Iguana-Tweaks.zs:31]: This recipe seems to be wrong or not used.

image
 This is the one we use

Comment (repo-alt) [reply scripts/Iguana-Tweaks.zs:31]: Just unwrap one line below in github GUI

**Review by repo-alt — APPROVED** [review]

File: scripts/Iguana-Tweaks.zs (hunk 1 above)
Comment (D-Cysteine) [reply scripts/Iguana-Tweaks.zs:31]: Isn't that the same as the one in this file? 5x clay dust:
```
[null, null, null],
[<ore:dustClay>, null, <ore:dustClay>],
[<ore:dustClay>, <ore:dustClay>, <ore:dustClay>]]);
```

**Comment by Dream-Master** [comment]
ok my bad i dint see the thrid line )

**Pull request merged**